docs/api/v1beta2/image-automation.md

```diff
@@ -735,7 +735,7 @@ github.com/fluxcd/pkg/apis/meta.ReconcileRequestStatus
 </div>
 </div>
 <h3 id="image.toolkit.fluxcd.io/v1beta2.ObservedPolicies">ObservedPolicies
-(<code>map[string]./api/v1beta2.ImageRef</code> alias)</h3>
+(<code>map[string]github.com/fluxcd/image-automation-controller/api/v1beta2.ImageRef</code> alias)</h3>
```

Comment:
Strange, I normally don't see changes in constants affecting this automatically generated file. Was this change result of running `make` or `make api-docs`? If yes, all good.

Reply:
This change is the result of running ‎⁠make test⁠, which triggers ‎⁠make api-docs⁠ as part of the process.

Reply:
Maybe a mismatch in the kubebuilder versions